projects/packages/a8c-mc-stats/tests/php/StatsTest.php

```diff
@@ -1,4 +1,4 @@
-<?php // phpcs:ignore Squiz.Commenting.FileComment.Missing
+<?php
```

Comment:
In some places we're doing `*Test.php` and in others `*_Test.php` (with a leading underscore). Is this intentional? The PR description specifically mentions doing the latter for Jetpack-the-plugin.

Reply:
Oops, too much copy-pasting. :sweat_smile: 

For the most part, if the class name already has underscores I did `_Test`, while if it was CamelCase I'd do just `Test`.

Reply:
Looking closer, it looks like all the ones with no-underscore `Test` were already named that way in the first place, so all I did was rename the file to match.